packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -284,6 +284,11 @@ export class TestFixture {
 
     return this;
   }
+  public waitForComponentHydration(componentTag: string) {
+    cy.document()
+      .find(componentTag, {timeout: 10e3, includeShadowDom: true})
+      .should('have.class', 'hydrated');
```

Comment:
in other words: I ain't moving till the component is cooked and ready, you have 10 seconds.

Reply:
Not really: for example, sometimes we'll intercept the tag (nice), but it won't be hydrated: you can see that, for example, when "you didn't put this cmp in a cmp that can hold it, unacceptable."

That was one of my first guess, but we can't do it that easily, but we should probably add it as we go for "green paths" tests

Reply:
I guess we could maybe add a bool for that... I'll create a separate task for that.